### PR TITLE
Individual addresses balance in /wallet/balance

### LINKED ISSUE #1462: Include individual address balances in /balance and /wallet/balance response
The responses should be modified to add a new field,

 ```
"addresses": { "addr1": { balance_data }`
```

### PR DESCRIPTION
**Pull request by therealssj**
Fixes #1462 

Changes:
- `/wallet/balance/` and `/balance` now return individual address balances under `addresses` field.
- updated tests

Does this change need to mentioned in CHANGELOG.md?
Yes

### CODE REVIEW

**Comment by gz-c** [comment]
Can we do it for `/balance` too?

File: src/visor/visor.go [hunk 1]
```diff
@@ -1039,8 +1081,8 @@ func (vs *Visor) CreateWallet(wltName string, opts wallet.Options) (*wallet.Wall
 	return vs.Wallets.CreateWallet(wltName, opts, vs)
 }
 
-// GetBalanceOfAddrs returns balance pairs of given addreses
-func (vs Visor) GetBalanceOfAddrs(addrs []cipher.Address) ([]wallet.BalancePair, error) {
+// GetBalanceOfAddrs returns balance pairs of given addresses
+func (vs Visor) GetBalanceOfAddrs(addrs []cipher.Address) (wallet.AddressBalance, error) {
```
Comment (gz-c) [inline src/visor/visor.go]: Don't change this one

File: src/wallet/balance.go [hunk 2]
```diff
@@ -21,6 +21,15 @@ type Balance struct {
 	Hours uint64 `json:"hours"`
 }
 
+// BalanceResult contains the overall balance as well as of individual addresses
```
Comment (gz-c) [inline src/wallet/balance.go]: These changes aren't needed in wallet package, the structs can be in the gui package

File: src/visor/visor.go [hunk 3]
```diff
@@ -974,6 +974,48 @@ func (vs *Visor) AddressBalance(auxs coin.AddressUxOuts) (uint64, uint64, error)
 	return coins, hours, nil
 }
 
+// WalletBalance computes the total balance of a wallet
+func (vs *Visor) WalletBalance(wltID string) (wallet.BalanceResult, error) {
```
Comment (gz-c) [inline src/visor/visor.go]: This function should be on daemon.Gateway and use strand()

**Comment by therealssj** [comment]
@gz-c done. 
Now `GetWalletBalance` returns a sum of balances and a list of address balances

**Comment by gz-c** [comment]
#1440 merged, need to fix conflicts

**Comment by therealssj** [comment]
@gz-c resolved

**Comment by therealssj** [comment]
@gz-c done.
I changed `createWallet` to accept another parameter `seed` and hardcoded a seed for `TestStableWalletBalance`

in `TestLiveWalletBalance` I am just checking that `addresses` is not null

**Pull request merged**